src/ports/catalog/queries.ts

```diff
@@ -0,0 +1,355 @@
+      : undefined,
+    filters.emoteCategory
+      ? getEmoteCategoryWhere(filters.emoteCategory)
+      : undefined,
+    filters.emotePlayMode?.length
+      ? getEmotePlayModeWhere(filters.emotePlayMode)
+      : undefined,
+    filters.contractAddresses?.length
+      ? getContractAddressWhere(filters.contractAddresses)
+      : undefined,
+    filters.minPrice ? getMinPriceWhere(filters.minPrice) : undefined,
+    filters.maxPrice ? getMaxPriceWhere(filters.maxPrice) : undefined,
+    filters.onlyListing ? getOnlyListingsWhere() : undefined,
+    filters.onlyMinting ? getOnlyMintingWhere() : undefined,
+  ].filter(Boolean)
+
+  if (!conditions.length) {
+    return SQL``
+  }
+
+  const result = SQL`WHERE `
+  conditions.forEach((condition, index) => {
+    if (condition) {
+      result.append(condition)
+      if (conditions[index + 1]) {
+        result.append(SQL` AND `)
+      }
+    }
+  })
+
+  return result.append(` `)
+}
+
+export const getCollectionsItemsCatalogQuery = (
+  schemaVersion: string,
+  filters: CatalogQueryFilters
+) => {
+  const query = SQL`
+            SELECT
+              items.id, 
+              to_json(
+                CASE WHEN (
+                  items.item_type = 'wearable_v1' OR items.item_type = 'wearable_v2' OR items.item_type = 'smart_wearable_v1') THEN metadata_wearable 
+                  ELSE metadata_emote 
+                END
+              ) as metadata,
+              items.image, 
+              items.blockchain_id,
+              items.collection,
+              items.rarity,
+              items.item_type::text,
+              items.price,
+              items.available,
+              items.search_is_store_minter,
+              items.creator,
+              items.beneficiary,
+              items.created_at,
+              items.updated_at,
+              items.reviewed_at,
+              items.sold_at,
+              ${filters.network} as network,
+              items.first_listed_at,
+              nfts.min_price AS min_listing_price,
+              nfts.max_price AS max_listing_price, 
+              nfts.listings_count as listings_count,
+              nfts.owners_count as owners_count,
+              CASE
+                WHEN items.available > 0 THEN LEAST(items.price, nfts.min_price) 
+                ELSE nfts.min_price 
+              END AS min_price,
+              CASE 
+                WHEN available > 0 THEN GREATEST(items.price, nfts.max_price) 
+                ELSE nfts.max_price END
+             AS max_price
+            FROM `
+    .append(schemaVersion)
+    .append(
+      `.item_active AS items 
+            LEFT JOIN (
+              SELECT 
+                nft.item, 
+                COUNT(orders.id) AS listings_count,
+                MIN(orders.price) AS min_price,
+                MAX(orders.price) AS max_price,
+                COUNT(DISTINCT nft.owner) as owners_count
+                MAX(orders.updated_at) AS max_order_created_at
```

Comment:
This statement uses the column `updated_at` and renames it as `max_order_created_at `. Is this on purpose, or is it an error?